doc/api/cli.md

```diff
@@ -336,6 +343,64 @@ file will be created if it does not exist, and will be appended to if it does.
 If an error occurs while attempting to write the warning to the file, the
 warning will be written to stderr instead.
 
+### `--diagnostic-report-directory`
+<!-- YAML
+added: REPLACEME
+-->
+
+Location at which the report will be generated.
+
+### `--diagnostic-report-filename`
+<!-- YAML
+added: REPLACEME
+-->
+
+Name of the file to which the report will be written.
+
+### `--diagnostic-report-on-fatalerror`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables the report to be triggered on fatal errors (internal errors within
+the Node.js runtime such as out of memory) that lead to termination of the
+application, if `--experimental-report` is enabled. Useful to inspect various
+diagnostic data elements such as heap, stack, event loop state, resource
+consumption etc. to reason about the fatal error.
+
+### `--diagnostic-report-on-signal`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables report to be generated upon receiving the specified (or predefined)
+signal to the running Node process, if `--experimental-report` is enabled.
+The signal to trigger the report is specified through `--diagnostic-report-signal`.
+
+### `--diagnostic-report-signal=signal`
+<!-- YAML
+added: REPLACEME
+-->
+
+Sets or resets the signal for report generation (not supported on windows).
+Default signal is `SIGUSR2`.
+
+### `--diagnostic-report-uncaught-exception`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enables report to be generated on un-caught exceptions, if
+`--experimental-report` is enabled. Useful when inspecting JavaScript stack in
+conjunction with native stack and other runtime environment data.
+
+### `--diagnostic-report-verbose`
```

Comment:
As the last example in the `report.md` mentions `--diagnostic-report-verbose=true`, should this be `--diagnostic-report-verbose=true|false`?